### PR TITLE
Feath

### PR DESCRIPTION
**Pull request by kmusial1988**

### CODE REVIEW

File: src/main/java/pl/sda/weather/controllers/ControllersLocation.java [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+import pl.sda.weather.service.ILocationService;
+import pl.sda.weather.service.impl.LocationServiceImpl;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.UUID;
+
+import static pl.sda.weather.Gui.showMainMenu;
+
 public class ControllersLocation {
```
Comment (MichalOrlowski) [inline src/main/java/pl/sda/weather/controllers/ControllersLocation.java]: should be LocationController (proper naming convention)

File: src/main/java/pl/sda/weather/controllers/ControllersLocation.java [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+
+        showMainMenu();
+
+    }
+
+    public void displayLocation() {
+
+        List<LocationModel> list = locationService.getLocationModelFromFile();
+
+        for (LocationModel locationModel : list) {
```
Comment (MichalOrlowski) [inline src/main/java/pl/sda/weather/controllers/ControllersLocation.java]: maybe use stream? :)

File: src/main/java/pl/sda/weather/dao/ILocationDAO.java [hunk 3]
```diff
@@ -1,9 +1,15 @@
 package pl.sda.weather.dao;
 
 
+import pl.sda.weather.model.LocationModel;
+
+import java.util.List;
 
 public interface ILocationDAO {
```
Comment (MichalOrlowski) [inline src/main/java/pl/sda/weather/dao/ILocationDAO.java:8]: I think better naming will be "Repository". DAO = data access object (and should be object, not interface)

File: src/main/java/pl/sda/weather/dao/ILocationDAO.java [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
 
 
+    void addLocationModelToDB(LocationModel locationModel);
```
Comment (MichalOrlowski) [inline src/main/java/pl/sda/weather/dao/ILocationDAO.java:11]: maybe rename to "save model" -> interface shouldn't know terms like "file, database etc."

**Review by Wielopol — APPROVED** [review]

**Pull request merged**